Dockerfile

```diff
@@ -1,12 +1,25 @@
-FROM golang:alpine
+FROM ubuntu:14.04 
 MAINTAINER Michal Rostecki <mrostecki@mirantis.com>
 
-RUN apk --no-cache add \
-	alpine-sdk \
+RUN apt-get update && apt-get install -y software-properties-common
+RUN add-apt-repository ppa:ubuntu-lxc/lxd-stable
+RUN apt-get update && apt-get install -y golang
+
+ENV GOPATH /go
+ENV PATH $GOPATH/bin:/usr/local/go/bin:$PATH
+
+RUN mkdir -p "$GOPATH/src" "$GOPATH/bin" && chmod -R 777 "$GOPATH"
```

Comment:
I guided with official dockerfile for golang https://github.com/docker-library/golang/blob/3a3e91c242b58a7d4e6022b3710b2e871f0ee5d6/1.7/Dockerfile. So I suppose it's better to keep it as 777.